### DISCUSSION TITLE
Hiding specialists

### DISCUSSION THREAD

**Discussion by Maksimov112358** [fedorovvl/tso_client#204] [Ideas]
Hello!

Would be great to have a possibility to hide geologists not only from Star Menu and hide them from Specialist's List (F4) as well.

Thanks anyway!

**Comment by fedorovvl** [comment]
done in https://github.com/fedorovvl/tso_client/commit/ee146630d84b4ac0e910b8f27272e29705d516ab
wait next release

**Reply by Maksimov112358** [reply]
Much appreciate!
